### ISSUE TITLE
Cleanup/rename antibiotics_family column in combined metadata table

### ISSUE THREAD

**Issue by cmirzayi** [waldronlab/curatedMetagenomicDataCuration#51]
**Is your feature request related to a problem? Please describe.**
The `antibiotics_family` column in the combined metadata table contains information on medications and treatments that are not antibiotics (antivirals, beta blockers, blood pressure meds, diabetes meds, etc.). While it's not an issue to filter these non-antibiotic treatments out myself, this could potentially cause confusion for users who would not know to look at this column for general medication information.

**Describe the solution you'd like**
Rename this column to better reflect what it actually is: medications.

You could also split medications and antibiotics into separate columns, since antibiotics probably have a stronger impact on the microbiome compared to most other medications. This would ensure backward compatibility with the antibiotics_family column while properly labeling medications.

**Comment by lwaldron** [comment]
@cmirzayi would you be willing to split medications and antibiotics_family into separate columns? I think the issue is unawareness of curators about what these medications are, so your help would be valuable. @paolinomanghi FYI.

**Comment by paolinomanghi** [comment]
Should I correct all the metadata tables singularly? Is not big deal, for
me.

**Comment by lwaldron** [comment]
Yes, in each metadata table individually. I hope it's not too much of a
burden or can be done with scripts.

**Comment by paolinomanghi** [comment]
No it can be done easily.

**Comment by lwaldron** [comment]
thanks @paolinomanghi. You should work with with @cmirzayi who is studying antibiotics specifically and can help to get it right.

**Comment by lwaldron** [comment]
Pinging @cmirzayi, would be great to resolve this for the Bioconductor 3.15 release.
